pipelines/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md

```diff
@@ -1,3 +1,12 @@
+# MultiSampleSmartSeq2_v2.0.0
+
+2019-06-04 (Date of Last Commit)
+* Removed zarr output and made loom output as default
+
+* Loom file attribute names have changed: CellID: cell_names, Gene: gene_names and Accession: ensembl_ids
+
+* Loom file name has changed from out.loom to "plateid".loom
+
```

Comment:
Add the expected counts output in loom to the changelog please

Reply:
It looks like it was in the previous release on 5/7 changelog, but we should update it from estimated -> expected counts to be consistent?

Reply:
Good point!

Reply:
Done